lib/fluent/plugin/parser_csv.rb

```diff
@@ -23,8 +23,10 @@ module Plugin
     class CSVParser < ValuesParser
       Plugin.register_parser('csv', self)
 
+      config_param :delimiter, :string, default: ','
+
       def parse(text)
-        yield values_map(CSV.parse_line(text))
+        yield values_map(CSV.parse_line(text,{col_sep: @delimiter}))
```

Comment:
Use keyword argument style instead of specifying a hash object: `CSV.parse_line(text, col_sep: @delimiter)`